.github/workflows/formatter.yaml

```diff
@@ -0,0 +1,16 @@
+name: formatter
+on: [push, pull_request]
```

Comment:
is it really necessary to trigger the formatter on *push*? I mean, the trigger on *pull_request* should run the job again, if you push into a pending PR, isn't it?

Reply:
Actually, no. But I thought it would not hurt.

Reply:
I changed it to `pull_request` only.